app/helpers/harvests_helper.rb

```diff
@@ -20,20 +20,20 @@ def display_human_quantity(harvest)
       if harvest.unit == 'individual' # just the number
         number_to_human(harvest.quantity, strip_insignificant_zeros: true)
       elsif ! harvest.unit.blank? # pluralize anything else
-        return pluralize(number_to_human(harvest.quantity, strip_insignificant_zeros: true), harvest.unit)
+        pluralize(number_to_human(harvest.quantity, strip_insignificant_zeros: true), harvest.unit)
```

Comment:
With these if-elsif-else and returns; we should probably try to return early for the error conditions/edge cases
```
def foo()
    return nil if impossible?
    return nil if boring_dont_care_about?

    pluralize(number_to_human(harvest.quantity, strip_insignificant_zeros: true), harvest.unit)
end
```

If find that cleans up a lot of the intent of the method